### PR TITLE
Add an option to use a placeholder for the freedesktop xml file

### LINKED ISSUE #118: Allow installation of this gem on systems without freedesktop.xml
A lot of projects have this gem as a dependency, but are not actually using it. The post-install checks in the newest versions of this script make it impossible to build these projects, without providing a freedesktop.xml file.

This can be a problem for multiple reasons:
* If your project is providing a docker image, then your project might be forced to deal with the licensing issues that were solved in the latest versions of this gem
* Your operating system might not provide the right version of this file, forcing you to include a file that you download manually from the internet
* A user that is fairly new to ruby and tries to install Rails will have trouble installing a basic development environment: all basic tutorials will fail, and changes are that troubleshooting will scare off such a user.

Is it possible to provide some means to skip the installation part, and just install this gem without activating it?

A couple of suggestions:
* Provide an environment variable that people can use to indicate that they do not wish to use any freedesktop.xml
* Provide a dummy freedesktop.xml that conforms to the format that this gem needs, but does not have any actual content

### PR DESCRIPTION
**Pull request by Scharrels**
Fixes #118

### CODE REVIEW

**Comment by Scharrels** [comment]
Fixed #118

File: ext/mimemagic/Rakefile [hunk 1]
```diff
@@ -17,12 +17,24 @@ def locate_mime_database
    Ensure you have either installed the shared-mime-info package for your distribution, or
    obtain a version of freedesktop.org.xml and set FREEDESKTOP_MIME_TYPES_PATH to the location
    of that file.
+
+   This gem might be installed as a dependency of some bigger package, such as rails, activestorage,
+   axlsx or cucumber. While most of these packages use the functionality of ths gem, some gems have
```
Comment (herwinw) [inline ext/mimemagic/Rakefile:22]: `ths` => `this`

**Review by herwinw — CHANGES_REQUESTED** [review]

File: ext/mimemagic/Rakefile (hunk 1 above)
Comment (Scharrels) [reply ext/mimemagic/Rakefile:22]: Fixed.

**Comment by jellybob** [comment]
Thanks, this is now merged into 0.3.10

**Pull request closed without merging**